packages/api-fetch/src/index.js

```diff
@@ -135,14 +170,18 @@ function apiFetch( options ) {
 		}
 
 		// If the nonce is invalid, refresh it and try again.
-		return window
-			.fetch( apiFetch.nonceEndpoint )
-			.then( checkStatus )
-			.then( ( data ) => data.text() )
-			.then( ( text ) => {
-				apiFetch.nonceMiddleware.nonce = text;
-				return apiFetch( options );
-			} );
+		return (
+			window
+				// @ts-ignore
```

Comment:
Were we going to type this one, too?

Reply:
No, I wasn't sure where these came from or how to type them effectively without having to declare the types for all the things that TS is doing a good job of inferring. How does the `nonceMiddleware` end up on `apiFetch`?

Reply:
I see. Well, this is a mess 😉 Should it have been a part of the nonce middleware instead? It's definitely something we can fix in another PR anyway.